### ISSUE TITLE
[FEATURE] Have the ability to disable the init container (which requires privileged permissions)

### ISSUE THREAD

**Issue by digger18** [kontena/akrobateo#30]
I would like to have the ability to disable the init container via environment variable or annotation, since my cluster security settings do not allow privileged containers.

I will make sure that ```/proc/sys/net/ipv4/ip_forward = 1``` myself.

Thanks in advance.

**Comment by HuxyUK** [comment]
Yes I have the same issue. I think also it would be nice to see a PodSecurityPolicy that allows you to easily roll out the load balancer when you have a restrictive cluster.
